Dockerfile

```diff
@@ -39,35 +38,46 @@ RUN yum install -y git \
 
 
 # various system deps
-RUN yum install -y epel-release \
- && yum install -y mysql-devel \
-                   Xvfb \
+RUN echo $'\n\
+[google-chrome]\n\
+name=google-chrome\n\
+baseurl=http://dl.google.com/linux/chrome/rpm/stable/x86_64\n\
+enabled=1\n\
+gpgcheck=1\n\
+gpgkey=https://dl-ssl.google.com/linux/linux_signing_key.pub' \
+ > /etc/yum.repos.d/google-chrome.repo \
+  && yum install -y epel-release \
+  && yum install -y mysql-devel \
                    firefox \
-                   qt5-qtwebkit-devel \
-                   libicu \
+                   google-chrome-stable \
                    unzip \
                    ImageMagick \
                    ImageMagick-devel \
                    pcre-devel \
                    openssl-devel \
                    libaio \
                    dbus \
-# sphinx deps + installation
                    postgresql-libs \
                    unixODBC \
   && yum clean all -y \
   && curl http://sphinxsearch.com/files/sphinx-2.2.11-1.rhel7.x86_64.rpm > /tmp/sphinx-2.2.11-1.rhel7.x86_64.rpm \
-  && yum install -y /tmp/sphinx-2.2.11-1.rhel7.x86_64.rpm
+  && yum install -y /tmp/sphinx-2.2.11-1.rhel7.x86_64.rpm \
+  && wget -N https://chromedriver.storage.googleapis.com/$(curl -sS chromedriver.storage.googleapis.com/LATEST_RELEASE)/chromedriver_linux64.zip -P /tmp \
+  && unzip /tmp/chromedriver_linux64.zip -d /tmp \
+  && rm /tmp/chromedriver_linux64.zip \
+  && mv -f /tmp/chromedriver /usr/local/bin/chromedriver \
+  && chown root:root /usr/local/bin/chromedriver \
+  && chmod 0755 /usr/local/bin/chromedriver \
+  && sed --in-place "s/databases 16/databases 32/" /etc/redis.conf \
```

Comment:
I don't think we still need this, right?

(related CI failure: https://circleci.com/gh/3scale/system-builder/15 )